Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git://github.com/huboard/warden-github.git
-  revision: 459903681fa00820065348c27639b9f27db9fa55
+  revision: 3c3cf6943b01ff12cfe22a1ad1202ddcc4721e66
```

Comment:
Yep, we have a dependency on so custom code in the warden-github gem to figure out what scopes we have requested. 

The Gemfile.lock specifies that we want that exact sha. 

So, our heroku build will pull that new sha on the next deploy